examples/torch/pearl_ml1_push.py

```diff
@@ -0,0 +1,135 @@
+"""PEARL ML1 example."""
+
+import os
+
+from metaworld.benchmarks import ML1
+
+from garage.envs import GarageEnv, normalize
+from garage.experiment import LocalRunner, SnapshotConfig, wrap_experiment
+from garage.experiment.deterministic import set_seed
+from garage.experiment.meta_evaluator import MetaEvaluator
+from garage.experiment.task_sampler import SetTaskSampler
+from garage.sampler import LocalSampler
+from garage.torch.algos import PEARL
+from garage.torch.algos.pearl import PEARLWorker
+from garage.torch.embeddings import MLPEncoder
+from garage.torch.policies import (ContextConditionedPolicy,
+                                   TanhGaussianMLPPolicy)
+from garage.torch.q_functions import ContinuousMLPQFunction
+import garage.torch.utils as tu
+
+algo_params = dict(
+    num_epochs=1000,
+    num_train_tasks=50,
+    num_test_tasks=10,
+    latent_size=7,
+    encoder_hidden_sizes=[200, 200, 200],
+    net_size=300,
+    meta_batch_size=16,
+    num_steps_per_epoch=4000,
+    num_initial_steps=4000,
+    num_tasks_sample=15,
+    num_steps_prior=750,
+    num_extra_rl_steps_posterior=750,
+    num_steps_per_eval=450,
+    batch_size=256,
+    embedding_batch_size=64,
+    embedding_mini_batch_size=64,
+    max_path_length=150,
+    reward_scale=10.,
+    use_information_bottleneck=True,
+    use_next_obs_in_context=False,
+    use_gpu=True,
+)
+
+
+# pylint: disable=unused-argument
+@wrap_experiment
+def torch_pearl_ml1_push(ctxt=None, seed=1, **params):
```

Comment:
please explicitly list the params to the function -- you can use defaults you have here as the default args. this is the style we're going for with the wrap_experiment API

Reply:
an alternative (if you want to avoid re-documenting a ton of parameters), is just to pass most of these to PEARL instead of making them parameters of the experiment.

Reply:
...or just don't pass them if they are the same as the default

Reply:
wrap_experiment throws an error if I don't pass ctxt and seed. It calls another method, and these attributes don't have default values.

Reply:
@krzentner can you help here?

Reply:
Yeah, `wrap_experiment` requires explicitly listing the parameters. That could be changed, but I haven't added support for it yet.

You shouldn't need to pass `seed`, but `ctxt` is required.

Reply:
If I don't pass in `seed`, I get this error:

```
File "/home/lindawong/Desktop/github/garage/src/garage/sampler/worker.py", line 140, in worker_init
    deterministic.set_seed(self._seed + self._worker_number)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int
```'

Reply:
Oh, did you still call `set_seed` below?

Reply:
No. I got the error above when I removed `seed` from the argument list. I removed `set_seed` as well.

Reply:
There's probably a bug in `wrap_experiment`. When other examples call `wrap_experiment`, they also pass in `seed`.